
git.savannah.gnu.org/cgit/freeipmi.git/commit/?id=ececf09

toolcommon: fix gcc15 compilation error
Albert Chu <chu11@llnl.gov>	2025-02-20 12:16:59 -0800


adapted to opensuse's source tree (ChangeLog)

--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,8 @@
+2025-02-20 Pavel Cahyna <pcahyna@redhat.com>
+
+	* common/toolcommon/tool-config-file-common.c: Fix gcc 15
+	compilation failure.
+
 2025-01-02 Albert Chu <chu11@llnl.gov>

 	* ipmi-sensors/ipmi-sensors-oem-intel-quanta-qssc-s4r.h: Fix header guard.
--- a/common/toolcommon/tool-config-file-common.c
+++ b/common/toolcommon/tool-config-file-common.c
@@ -78,14 +78,14 @@ _config_file_bool (conffile_t cf,
                    void *app_ptr,
                    int app_data)
 {
-  int *bool;
+  int *value;
 
   assert (data);
   assert (optionname);
   assert (option_ptr);
 
-  bool = (int *)option_ptr;
-  *bool = data->boolval;
+  value = (int *)option_ptr;
+  *value = data->boolval;
   return (0);
 }
 
